<ui version="4.0" >
 <class>qtadslWindow</class>
 <widget class="QMainWindow" name="qtadslWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QtAdsl</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="qtadslicons.qrc" >
    <normaloff>:/icons/icons/qtadsl.png</normaloff>:/icons/icons/qtadsl.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout" >
    <item row="2" column="0" >
     <widget class="QDialogButtonBox" name="buttons" >
      <property name="standardButtons" >
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QToolBox" name="toolBox" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="page" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>254</width>
         <height>130</height>
        </rect>
       </property>
       <attribute name="label" >
        <string>User settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="1" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Adsl user name:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLineEdit" name="nameLine" />
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Adsl password:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" >
         <widget class="QLineEdit" name="passwdLine" >
          <property name="echoMode" >
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="0" >
         <spacer name="verticalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0" >
         <spacer name="verticalSpacer_3" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>254</width>
         <height>130</height>
        </rect>
       </property>
       <attribute name="label" >
        <string>Advaced settings</string>
       </attribute>
       <widget class="QCheckBox" name="isAutomaticConnect" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>10</y>
          <width>211</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text" >
         <string>Automatic connect...</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>272</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionConnect" />
    <addaction name="actionDisconnect" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionHelp" />
    <addaction name="actionAbout" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuHelp" />
  </widget>
  <action name="actionConnect" >
   <property name="icon" >
    <iconset resource="qtadslicons.qrc" >
     <normaloff>:/connection/icons/connect.png</normaloff>:/connection/icons/connect.png</iconset>
   </property>
   <property name="text" >
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect" >
   <property name="icon" >
    <iconset resource="qtadslicons.qrc" >
     <normaloff>:/connection/icons/disconnect.png</normaloff>:/connection/icons/disconnect.png</iconset>
   </property>
   <property name="text" >
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionSettings" >
   <property name="icon" >
    <iconset resource="qtadslicons.qrc" >
     <normaloff>:/icons/icons/preferences_system.png</normaloff>:/icons/icons/preferences_system.png</iconset>
   </property>
   <property name="text" >
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="icon" >
    <iconset resource="qtadslicons.qrc" >
     <normaloff>:/icons/icons/exit.png</normaloff>:/icons/icons/exit.png</iconset>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelp" >
   <property name="icon" >
    <iconset resource="qtadslicons.qrc" >
     <normaloff>:/icons/icons/help.png</normaloff>:/icons/icons/help.png</iconset>
   </property>
   <property name="text" >
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="icon" >
    <iconset resource="qtadslicons.qrc" >
     <normaloff>:/icons/icons/qtadsl.png</normaloff>:/icons/icons/qtadsl.png</iconset>
   </property>
   <property name="text" >
    <string>About QtAdsl</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qtadslicons.qrc" />
 </resources>
 <connections/>
</ui>
